Resources/Textures/SimpleStation14/Mobs/Ghosts/ghost_admin.rsi/meta.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Made by Pspritechologist?",
```

Comment:
I don't remember who made this, is this what the original copyright field says?

Reply:
Yes, I copied over the original meta file

Reply:
Hilarious, I'll ask @Pspritechologist